src/main/java/org/folio/processing/events/services/publisher/RestEventPublisher.java

```diff
@@ -36,19 +42,27 @@
           .withEventTTL(1)
           .withPublishedBy(PomReaderUtil.INSTANCE.constructModuleVersionAndVersion(PomReaderUtil.INSTANCE.getModuleName(), PomReaderUtil.INSTANCE.getVersion())));
 
-      PubSubClientUtils.sendEventMessage(event, params).whenComplete((published, throwable) -> {
-        if (throwable != null) {
-          LOGGER.error("Error publishing {} event", event.getEventType());
-          future.completeExceptionally(throwable);
-        } else {
-          LOGGER.info("{} event has been published", event.getEventType());
-          future.complete(event);
-        }
-      });
+      RestUtil.doRequest(params, "/pubsub/publish", HttpMethod.POST, event)
```

Comment:
So we still need endpoints from pubsub library?

Reply:
Following good practices, we need to annotate Deprecated in this release and remove this functionality in the next release.